azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -31,7 +31,7 @@ class IonQBackend(AzureBackend):
 
     @classmethod
     def _default_options(cls):
-        return Options(shots=500)
+        return Options(shots=500, targetCapability="ionq")
```

Comment:
Is there a reason for specifying the `targetCapability` by default? I believe the previous behavior should still be the default.

Reply:
The behavior is the same by default. Only if someone sets targetCapability to "BasicExecution" is the qir translation triggered.

Reply:
I've removed the default targetCapability for ionq since it was causing confusion.